.changeset/pretty-zebras-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': major
```

Comment:
should be `patch`, not `major`.
```suggestion
'@itwin/itwinui-react': patch
```

when you run `yarn changeset`, it first asks you "which packages have MAJOR changes?" you can skip that by pressing <kbd>Enter</kbd> (without selecting anything with <kbd>Space</kbd>).